### PR TITLE
Replace all references to storage.googleapis.com/kubernetes-release with dl.k8s.io

### PR DESCRIPTION
**Pull request by 9bany**
fixes: https://github.com/kubernetes/website/issues/41069

### CODE REVIEW

**Comment by T-Lakshmi** [comment]
Hi @9bany,
Thanks for the PR contribution.
Seems this PR has changes to multiple languages. In general we are not supposed to submit a PR to multiple languages. I recommend you to break this up into multiple PRs, where we have a single PR per language.

**Comment by 9bany** [comment]
> Hi @9bany, Thanks for the PR contribution. Seems this PR has changes to multiple languages. In general we are not supposed to submit a PR to multiple languages. I recommend you to break this up into multiple PRs, where we have a single PR per language.

Let me break this up into multiple PRs, thanks @T-Lakshmi

**Review by dipesh-rawat — COMMENTED** [review]
Usually, we do not make changes to blog posts that are more than a year old. However, this PR proposes beneficial modifications regarding the use of `dl.k8s.io` as the kubernetes release artifact download location. I'll lgtm for now and ask a blog approver to review it.
/lgtm

@kubernetes/sig-docs-blog-owners , Please take a look.

**Comment by 9bany** [comment]
Any problems in this pull request ? @-maintainers @-reviewers

File: content/en/blog/_posts/2019-03-28-running-kubernetes-locally-on-linux-with-minikube.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ In order to manage the Kubernetes cluster, we need to install [kubectl](https://
 The recommended way to install it on Linux is to download the pre-built binary and move it to a directory under the `$PATH`.
 
 ```shell
-curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl \
+curl -LO https://dl.k8s.io/release/$(curl -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl \
```
Comment (sftim) [inline content/en/blog/_posts/2019-03-28-running-kubernetes-locally-on-linux-with-minikube.md:83]:
```suggestion
curl -LO https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl \
```

File: content/en/blog/_posts/2015-12-00-Creating-Raspberry-Pi-Cluster-Running.md [hunk 2]
```diff
@@ -85,7 +85,7 @@ f017f405ff4b        gcr.io/google\_containers/hyperkube-arm:v1.1.2   "/hyperkube
 
 When that’s looking good we’re able to access the master node of the Kubernetes cluster with kubectl. Kubectl for ARM can be downloaded from googleapis storage. kubectl get nodes shows which cluster nodes are registered with its status. The master node is named 127.0.0.1.
 ```
-$ curl -fsSL -o /usr/bin/kubectl https://storage.googleapis.com/kubernetes-release/release/v1.1.2/bin/linux/arm/kubectl
+$ curl -fsSL -o /usr/bin/kubectl https://dl.k8s.io/release/v1.1.2/bin/linux/arm/kubectl
```
Comment (sftim) [inline content/en/blog/_posts/2015-12-00-Creating-Raspberry-Pi-Cluster-Running.md:88]:
```suggestion
$ curl -fsSL -L -o /usr/bin/kubectl https://dl.k8s.io/release/v1.1.2/bin/linux/arm/kubectl
```

Also acceptable: leave this as it is (we don't maintain old blog articles)

**Review by sftim — COMMENTED** [review]
Hi. These changes won't work without modifications.

**Comment by sftim** [comment]
/retitle Replace all references to storage.googleapis.com/kubernetes-release with dl.k8s.io

File: content/en/blog/_posts/2015-12-00-Creating-Raspberry-Pi-Cluster-Running.md [hunk 3]
```diff
@@ -85,7 +85,7 @@ f017f405ff4b        gcr.io/google\_containers/hyperkube-arm:v1.1.2   "/hyperkube
 
 When that’s looking good we’re able to access the master node of the Kubernetes cluster with kubectl. Kubectl for ARM can be downloaded from googleapis storage. kubectl get nodes shows which cluster nodes are registered with its status. The master node is named 127.0.0.1.
 ```
-$ curl -fsSL -o /usr/bin/kubectl https://storage.googleapis.com/kubernetes-release/release/v1.1.2/bin/linux/arm/kubectl
+$ curl -fsSL -L -o /usr/bin/kubectl https://dl.k8s.io/release/v1.1.2/bin/linux/arm/kubectl
```
Comment (sftim) [inline content/en/blog/_posts/2015-12-00-Creating-Raspberry-Pi-Cluster-Running.md:88]: Ah sorry, the `-L` was already there.
```suggestion
$ curl -fsSL -o /usr/bin/kubectl https://dl.k8s.io/release/v1.1.2/bin/linux/arm/kubectl
```

**Review by sftim — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**